server/src/main/java/org/elasticsearch/index/fielddata/GeoPointScriptDocValues.java

```diff
@@ -13,6 +13,7 @@
 import org.elasticsearch.script.GeoPointFieldScript;
 
 public final class GeoPointScriptDocValues extends AbstractSortedNumericDocValues {
+    // private final AbstractPointFieldScript<GeoPoint> script;
```

Comment:
can we remove this?

Reply:
I believe I did that in the same commit that removed the script classes.